src/core/ext/filters/channel_idle/channel_idle_filter.cc

```diff
@@ -277,9 +278,10 @@ void ChannelIdleFilter::StartIdleTimer() {
 
 void ChannelIdleFilter::CloseChannel() {
   auto* op = grpc_make_transport_op(nullptr);
-  op->disconnect_with_error = grpc_error_set_int(
-      GRPC_ERROR_CREATE("enter idle"),
-      StatusIntProperty::ChannelConnectivityState, GRPC_CHANNEL_IDLE);
+  op->disconnect_with_error =
+      GRPC_ERROR_BUILDER(kUnknown, "enter idle")
+          .Set(StatusIntProperty::ChannelConnectivityState, GRPC_CHANNEL_IDLE)
+          .build();
```

Comment:
`.Build()`

Reply:
Yep. Also we can consider to have an implicit converter to absl::Status to be able to omit Build() function call.